pysatModels/utils/extract.py

```diff
@@ -442,42 +441,43 @@ def instrument_view_irregular_model(inst, model, inst_name, mod_name,
     model : pysat.Instrument
         Xarray pysat Instrument with model data that will be interpolated onto
         the `inst` locations
-    inst_name : array-like
+    inst_name : list
         List of variable names containing the instrument data coordinates
         at which the model data will be interpolated. Do not include 'time',
         only spatial coordinates. Same ordering as used by mod_name.
-    mod_name : array-like
+    mod_name : list
         list of names of the data series to use for determining model locations
         in the same order as inst_name.  These must make up a regular grid.
     mod_datetime_name : str
         Name of the data series in the model Dataset containing datetime info
     mod_units : list
-        units for each of the mod_name location attributes.  Currently
+        units for each of the mod_name location attributes. Units for
+        `mod_irreg_var' should replace those for `mod_reg_dim`. Currently
```

Comment:
The units for `mod_irreg_var` should be used instead of the actual units of `mod_reg_dim`, for the dimension that is going to be replaced by `mod_irreg_var`.

Reply:
By "should" do you mean "will" or "will, if included"?

Reply:
Updated docstring to reflect that users must provide the units.